doc/classes/VisualServer.xml

```diff
@@ -1397,6 +1406,9 @@
 			<return type="RID">
 			</return>
 			<description>
+				Creates a gi probe and adds it to the VisualServer. It can be accessed with the RID that is returned. This RID will be used in all [code]gi_probe_*[/code] VisualServer functions.
```

Comment:
"gi" should be capitalized ("GI probe"), as it's an acronym (well, initialism, but let's not be pedantic :P).